### PR TITLE
Updating to include django 2.1

### PR DESCRIPTION
**Pull request by jtrain**

### CODE REVIEW

**Comment by Anton-Shutik** [comment]
@jtrain Could you pls resolve conflicts ?

File: explorer/utils.py [hunk 1]
```diff
@@ -2,12 +2,7 @@
 import re
 
 from django.contrib.auth.forms import AuthenticationForm
-
-try:
-    from django.contrib.auth.views import login
-except ImportError:
-    pass
-
+from django.contrib.auth.views import LoginView
```
Comment (Anton-Shutik) [inline explorer/utils.py]: That breaks backwards compatibility right ?
Can we make the code working on previous django version too?

Comment (jtrain) [reply explorer/utils.py]: It was introduced django 1.11 earlier versions of django aren't supported by django foundation anymore.

Comment (jtrain) [reply explorer/utils.py]: https://docs.djangoproject.com/en/1.11/topics/auth/default/#django.contrib.auth.views.login

Comment (Anton-Shutik) [reply explorer/utils.py]: Cool, can we make tests passing by removing deprecated django versions ?

**Comment by jtrain** [comment]
I have cleaned the documentation/requirements/setup.py file for new supported versions >=1.11,<2.2

We use sql-explorer in Django 2.2 already, but tests fail. I will submit fixes as a separate PR for 2.2 support.

**Comment by Anton-Shutik** [comment]
@jtrain Thanks so much for you contribution! 
So, we should _either_ fix tests and rename PR with "Updating to include django 2.1-2.2" _or_ just drop django 2.2 support and add it in separate PR.

Just making sure we're merging PR that has all tests passing

**Pull request merged**